vue/components/ui/molecules/table/table.vue

```diff
@@ -1,7 +1,27 @@
 <template>
-    <table class="table" v-bind:class="classes" v-bind:style="style">
+    <table
+        class="table"
+        v-bind:class="{ classes, disableSelection: lastIndexWithShift !== null }"
+        v-bind:style="style"
+    >
+        <global-events
+            v-on:keydown.meta.65.exact.prevent="onCtrlA()"
+            v-on:keydown.ctrl.65.exact.prevent="onCtrlA()"
+            v-on:keydown.shift.up.exact="onShiftUp()"
+            v-on:keydown.shift.down.exact="onShiftDown()"
+            v-on:keydown.ctrl.alt.65.exact="onCtrlAltA()"
+            v-on:keydown.meta.alt.65.exact="onCtrlAltA()"
+        />
         <thead class="table-head">
             <tr>
+                <th class="checkboxes-th" v-if="enableCheckboxes">
+                    <checkbox

```

Comment:
Layout isn't aligned with the others:
<img width="552" alt="Screenshot 2020-01-27 at 15 18 14" src="https://user-images.githubusercontent.com/25725586/73186808-491ed780-4118-11ea-8926-7281c4eb24d0.png">